js/feature-detect.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env browser */
+const matchMedia = window.matchMedia || window.msMatchMedia;
+
+export default class FeatureDetect {
+  constructor() {
+    this.tests = {};
+  }
+
+  static mq(query) {
+    const mql = matchMedia(query);
```

Comment:
Check `matchMedia` is defined for IE9 and below, just so they don't error out?

Reply:
Or at the top fallback to a function that just returns false, as that's what we do after this.

Reply:
Good call, I definitely intended to do that but then forgot